File: runners/python3/sssa-create.py
# - Python - #

import json

import sys
sys.path.insert(0, './libs/sssa-python')
from SSSA import sssa

fi = open('./tmp/Python3_Create_Input.json', 'r')
input = fi.read()
args = json.loads(input)
fo = open('./tmp/Python3_Create_Output.json', 'w')
sss = sssa()
fo.write(json.dumps(sss.create(int(args[0]), int(args[1]), args[2].encode('utf-8'))))
fo.close()

File: runners/ruby/sssa-create.rb
# - Ruby - #

require 'json'
require_relative '../../libs/sssa-ruby/lib/sssa.rb'

input = File.read('./tmp/Ruby_Create_Input.json')
args = JSON.parse(input)
o = File.new('./tmp/Ruby_Create_Output.json', 'w')
o.write SSSA::create(args[0].to_i, args[1].to_i, args[2]).to_json
o.close
